public/app/features/alerting/unified/state/AlertingQueryRunner.ts

```diff
@@ -52,7 +53,14 @@ export class AlertingQueryRunner {
 
   async run(queries: AlertQuery[], condition: string) {
     const empty = initialState(queries, LoadingState.Done);
-    const queriesToRun = await this.prepareQueries(queries);
+    const error = initialState(queries, LoadingState.Error);
+    let queriesToRun: Array<AlertQuery<AlertDataQuery | ExpressionQuery>> = [];
+    try {
+      queriesToRun = await this.prepareQueries(queries);
```

Comment:
this was the only place i found to be able to catch the error, in upper levels, the error was not catch 🤷‍♀️